src/actions/gallery.js

```diff
@@ -0,0 +1,9 @@
+
+export const successGallery = (payload) => ({
```

Comment:
how exactly are you using these actions
you need to create services for api calls and then use actions in components to make api calls.
What is the purpose of this function ?